chart/kubeapps/Chart.yaml

```diff
@@ -1,21 +1,21 @@
 apiVersion: v1
 name: kubeapps
-version: 3.0.3
+version: 3.0.4
```

Comment:
please, revert this change, I plan to release 3.1.0. Bumping the version now will trigger a release but we are fixing a bug that should be added before.